src/common/techstack/techstack.jsx

```diff
@@ -1,10 +1,10 @@
-import { Tech } from ".";
+import { TechStackInfo } from ".";
 
```

Comment:
@Angryman18 My comment was about this file name.

Reply:
i guess it is done.